Rubberduck.Parsing/Symbols/ConcurrentlyConstructedDeclarationFinderFactory.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using Rubberduck.Parsing.Annotations;
+using Rubberduck.VBEditor.Application;
+
+namespace Rubberduck.Parsing.Symbols
+{
+    public class ConcurrentlyConstructedDeclarationFinderFactory : IDeclarationFinderFactory
```

Comment:
I know that Ninject does this. However, the two handwritten implementations are necessary for the API (`ParserState`) and the mocks (`MockParser` and a lot of tests explicitly newing up a `RubberduckParserState`).